src/sonic-config-engine/minigraph.py

```diff
@@ -191,6 +192,7 @@ def parse_dpg(dpg, hname):
             for addrtuple in pc_map.get(pcintfname, []):
                 pc_attributes.update(addrtuple)
                 pc_intfs.append(copy.deepcopy(pc_attributes))
+            pcs[pcintfname] = pc_attributes
```

Comment:
Do we want to keep 'members' (and 'vlanid') information duplicated in both minigraph_xxx_interfaces and minigraph_xxx? Or shall we let minigraph_xxx_interfaces to have only the same information with minigraph_interfaces?